include/userobjects/OpenMCUserObject.h

```diff
@@ -20,29 +20,19 @@
 
 #include "GeneralUserObject.h"
 
+// forward declarations
+class OpenMCProblemBase;
+
 /**
- * User object to modify the nuclides in an OpenMC tally.
+ * User object to modify an OpenMC tally
```

Comment:
I think this documentation is not referring to this class? This looks like a generic base class

Reply:
Good catch. Thanks!